code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -34,94 +34,501 @@
 /obj/item/slime_extract/pink
 	name = "pink slime extract"
 	icon_state = "pink slime extract"
 
+/obj/item/slime_extract/pink/activate(mob/living/carbon/human/user, datum/species/jelly/luminescent/species, activation_type)
+	switch(activation_type)
+		if(SLIME_ACTIVATE_MINOR)
+			if(user.gender != MALE && user.gender != FEMALE)
+				to_chat(user, "<span class='warning'>You can't swap your gender!</span>")
+				return
+
+			if(user.gender == MALE)
+				user.gender = FEMALE
+				user.visible_message("<span class='boldnotice'>[user] suddenly looks more feminine!</span>", "<span class='boldwarning'>You suddenly feel more feminine!</span>")
+			else
+				user.gender = MALE
+				user.visible_message("<span class='boldnotice'>[user] suddenly looks more masculine!</span>", "<span class='boldwarning'>You suddenly feel more masculine!</span>")
+			return 100
+
+		if(SLIME_ACTIVATE_MAJOR)
+			user.visible_message("<span class='warning'>[user]'s skin starts flashing hypnotically...</span>", "<span class='notice'>Your skin starts forming odd patterns, pacifying creatures around you.</span>")
+			for(var/mob/living/carbon/C in viewers(user, null))
+				if(C != user)
+					C.reagents.add_reagent("pax",2)
+			return 600
+
 /obj/item/slime_extract/green
 	name = "green slime extract"
 	icon_state = "green slime extract"
 
+/obj/item/slime_extract/green/activate(mob/living/carbon/human/user, datum/species/jelly/luminescent/species, activation_type)
+	switch(activation_type)
+		if(SLIME_ACTIVATE_MINOR)
+			to_chat(user, "<span class='warning'>You feel yourself reverting to human form...</span>")
+			if(do_after(user, 120, target = user))
+				to_chat(user, "<span class='warning'>You feel human again!</span>")
+				user.set_species(/datum/species/human)
+				return
+			to_chat(user, "<span class='notice'>You stop the transformation.</span>")
+
+		if(SLIME_ACTIVATE_MAJOR)
+			to_chat(user, "<span class='warning'>You feel yourself radically changing your slime type...</span>")
+			if(do_after(user, 120, target = user))
+				to_chat(user, "<span class='warning'>You feel different!</span>")
+				user.set_species(pick(/datum/species/jelly/slime, /datum/species/jelly/stargazer))
+				return
+			to_chat(user, "<span class='notice'>You stop the transformation.</span>")
+
 /obj/item/slime_extract/lightpink
 	name = "light pink slime extract"
 	icon_state = "light pink slime extract"
 
+/obj/item/slime_extract/lightpink/activate(mob/living/carbon/human/user, datum/species/jelly/luminescent/species, activation_type)
+	switch(activation_type)
+		if(SLIME_ACTIVATE_MINOR)
+			var/obj/item/slimepotion/docility/O = new(null, 1)
+			if(!user.put_in_active_hand(O))
+				O.forceMove(user.drop_location())
+			playsound(user, 'sound/effects/splat.ogg', 50, 1)
+			user.visible_message("<span class='warning'>[user] spits out [O]!</span>", "<span class='notice'>You spit out [O]!</span>")
+			return 150
+
+		if(SLIME_ACTIVATE_MAJOR)
+			var/obj/item/slimepotion/sentience/O = new(null, 1)
```

Comment:
Hugbox for people that dont want to work for potions by regular means.

Reply:
Fair, but i couldn't think of anything else for light pinks